src/fetching/putUndoEventResponse.ts

```diff
@@ -0,0 +1,27 @@
+import apiUrl from '../utils/apiUrl';
+
+import { ZetkinMembership } from '../types/zetkin';
+
+export default function putUndoEventResponse(event : number, org : number | undefined) : Promise<ZetkinMembership> {
+    const eventUndoSignup = async () => {
+        const mUrl = apiUrl('/users/me/memberships');
+        const mRes = await fetch(mUrl);
+        const mData = await mRes.json();
+        const orgMembership = mData.data.find((m : ZetkinMembership) => m.organization.id === org);
+
+        if (orgMembership) {
+            const url = apiUrl(`/orgs/${org}/actions/${event}/responses/${orgMembership.profile.id}`);
+            const eventRes = await fetch(url, {
+                method: 'DELETE',
+            });
+
+            if (eventRes.redirected === false) {
+                return null;
+            }
```

Comment:
What is the purpose of this?

Reply:
Since I had a return value on this function I got a parse-related error message from Next when clicking `Undo sign-up` twice, since this value is erased after the first click. After removing the return value per you comment below this is no longer an issue.